docker-compose-releng.yml

```diff
@@ -0,0 +1,84 @@
+version: "2.1"
+
+# TODO: what links do we need?
+services:
+  releng_db:
+    image: postgres:9.6
```

Comment:
For postgresql we should use the same setup as in nix setup. and run the same command as if we would be running from Nix. That way we avoid "it works on my here" situations.